Web/starter_project_iconics/pages/blog/singleBlog.tsx

```diff
@@ -0,0 +1,80 @@
+
+// type Blog  = {
+//     title : string,
+
+
+// }
+
+// interface props {
+//     title : Blog.title,
+
+// };
+
+const singleBlog = ({}) => {
+  return (
+    <div className="bg-white">
+        <h2 className="text-center blog-title pt-4">
+            The essential guide to Competitive Programming
+        </h2>
+        <p className="text-center uppercase blog-sub-title">
+            Programming, tech | 6 min Read
+        </p>
+        <div className="flex h-auto w-5/6  justify-center items-center mx-auto p-8">
+            <img src="/blogImg.png" className="mx-auto" alt="blog image"/>
```

Comment:
the alt should be specific to the image which means specific to the blog content (maybe the blog title or sth)